tests/test_backends.py

```diff
@@ -21,7 +21,11 @@
 def create_safetensors_or_bin_model_files(
     model_path: pathlib.Path, model_file_type: str, valid: bool
 ):
-    test_json_dict = {"a": 1, "b": 2}
+    test_json_dict = {
+        "a": 1,
+        "b": 2,
+        "quantization_config": {"quant_method": "bitsandbytes"},
+    }
     json_object = json.dumps(test_json_dict, indent=4)
 
     for file in ["tokenizer.json", "tokenizer_config.json"]:
```

Comment:
should this change also go into tokenizer config though? (I know it's just tests, but still...)